homeassistant/components/nextcloud/entity.py

```diff
@@ -12,19 +15,25 @@ class NextcloudEntity(CoordinatorEntity[NextcloudDataUpdateCoordinator]):
     """Base Nextcloud entity."""
 
     _attr_has_entity_name = True
-    _attr_icon = "mdi:cloud"
 
     def __init__(
-        self, coordinator: NextcloudDataUpdateCoordinator, item: str, entry: ConfigEntry
+        self,
+        coordinator: NextcloudDataUpdateCoordinator,
+        item: str,
+        entry: ConfigEntry,
+        desc: EntityDescription,
     ) -> None:
         """Initialize the Nextcloud sensor."""
         super().__init__(coordinator)
         self.item = item
         self._attr_translation_key = slugify(item)
```

Comment:
I know I implemented this, but back then it was the only way to implement translations as we didn't have entity descriptions. But can we maybe replace this with adding a translation key to every description? What do you think about this?

Reply:
i like this idea ... give me a moment 😁

Reply:
[b18a601](https://github.com/home-assistant/core/pull/94066/commits/b18a601ef805897d9f68cbd3dd47fcfff5fa667f)